src/IDriver.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * This file is part of Ares.
+ * Copyright (c) 2015 Václav Vrbka (http://aurielle.cz)
+ */
+
+namespace Grifart\Ares;
+
+
+interface IDriver
+{
+	/**
+	 * Fetches data of one subject identified by his IN from the ARES database.
+	 * @param string $in
+	 * @param bool $includeExpired
+	 * @return SubjectInfo
+	 */
+	function fetch($in, $includeExpired = FALSE);
```

Comment:
What is the point of `$includeExpired = FALSE`? Shouldn't it be better as parameter of Ares::findDetails()? I think you should recieve info of any subject that ever existed (if possible) and it is on the used to check in which state of life-cycle is company that you recieved info for.